.appveyor.yml

```diff
@@ -12,17 +12,16 @@ environment:
 init:
   - "ECHO %MINICONDA%"
   - C:\"Program Files (x86)"\"Microsoft Visual Studio 14.0"\VC\vcvarsall.bat %PLATFORM%
-  - ps: if($env:Platform -eq "x64"){Start-FileDownload 'http://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe' C:\Miniconda.exe; echo "Done"}
-  - ps: if($env:Platform -eq "x86"){Start-FileDownload 'http://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86.exe' C:\Miniconda.exe; echo "Done"}
+  - ps: Start-FileDownload 'http://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe' C:\Miniconda.exe
   - cmd: C:\Miniconda.exe /S /D=C:\xsimd-conda
   - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%MINICONDA%\\Library\\bin;%PATH%"
 
 install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
   - conda info -a
-  - conda install gtest cmake xtl==0.6.11 -c conda-forge
-  - cmake -G "NMake Makefiles" -D BUILD_TESTS=ON -D CMAKE_INSTALL_PREFIX=%MINICONDA%\\LIBRARY -D CMAKE_BUILD_TYPE=Release .
+  - conda install gtest cmake xtl==0.6.11 nlohmann_json -c conda-forge
+  - cmake -G "NMake Makefiles" -D DOWNLOAD_GTEST=ON -D CMAKE_INSTALL_PREFIX=%MINICONDA%\\LIBRARY -D CMAKE_BUILD_TYPE=Release .
```

Comment:
I think you should keep the `BUILD_TESTS=ON` instead of using `DOWNLOAD_GTEST=ON`. The build of `gtest` on Windows seems to produce static libraries by default.

Reply:
thx